lib/util/storage/log_output.dart

```diff
@@ -0,0 +1,41 @@
+import 'package:logger/logger.dart';
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:path_provider/path_provider.dart';
+
+/// Writes the log output to a file.
+class FileOutput extends LogOutput {
+  final bool overrideExisting;
+  final Encoding encoding;
```

Comment:
Should this variables also be defined as private?

```suggestion
  final bool _overrideExisting;
  final Encoding _encoding;
```